src/content/docs/ru/tutorial/0-introduction/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+type: tutorial
+unitTitle: 'Привет мир!'
+title: Создайте свой первый блог на Astro
+description: >-
+  Изучайте основы Астро с помощью проектного руководства. В этом учебнике вы найдете все необходимые знания для начала работы!
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+import Lede from '~/components/tutorial/Lede.astro';
+
+
+<Lede> В этом руководстве вы изучите ключевые функции Astro, создав полностью функциональный блог, от нуля до полноценного запуска! 🚀 </Lede>
+
+На этом пути вы:
```

Comment:
Can you please check these bullet points? They are not showing up as separate lines in the preview.